src/BookYourShow.Api/Controllers/LikeController.cs

```diff
@@ -0,0 +1,104 @@
+﻿using BookYourShow.Models;
+using BookYourShow.Repository;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BookYourShow.Controllers
+{
+    [Route("[controller]")]
```

Comment:
follow api standards mentioned in the doc

Reply:
Comments resolved